### PR TITLE
Create .gitattributes to ignore test folder

### PR DESCRIPTION
**Pull request by VincentLanglet**
HI @matthewelwell,

I noticed when installing this lib that the test folder was downloaded too.
This PR fix this.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.* export-ignore
```
[the remark this answers is not in the document]
Comment (VincentLanglet) [reply .gitattributes:1]: Sure, I personally use it here https://github.com/sonata-project/SonataAdminBundle/blob/4.x/.gitattributes#L5 and it works fine.

The idea is to ignore all the hidden files/folder like
```
.github/
.vscode/
.gitignore
.gitattributes
.php-cs-fixer.php
.prettierrc
```
but if you prefer we can list them explicitely

[the remark this answers is not in the document]
Comment (matthewelwell) [reply .gitattributes:1]: Ah, I see! I was reading it as a 'match all' condition in regex, but it's actually just matching any files with a leading `.`. Gotcha! 

Based on this confusion, however, I would rather see them listed explicitly if possible.

**Review by matthewelwell — APPROVED** [review]

**Comment by matthewelwell** [comment]
Thanks very much @VincentLanglet !

**Comment by matthewelwell** [comment]
Hey @VincentLanglet, I've just created v3.1.2 [here](https://github.com/Flagsmith/flagsmith-php-client/releases/tag/v3.1.2) but it's not yet been picked up by packagist it seems. Hopefully it will be there soon.

**Pull request merged**